pom.xml

```diff
@@ -110,7 +110,18 @@
       <version>${muleSdkApiVersion}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock</artifactId>
+      <version>1.58</version>
+      <scope>test</scope>
+      <exclusions>

```

Comment:
put this in depMagmt, and the dependency on each repo actually using it

Reply:
that is a refactor we probably want in master before the support branch. I agree that we should do it. But wouldn't it be cleaner if we leave this one as a plain, unchanged cherry pick (collection) and then do the refactor in master and cherry pick that one?